lib/wakatime_service.rb

```diff
@@ -87,7 +87,11 @@ def self.parse_user_agent(user_agent)
     else
       # Try parsing as browser user agent as fallback
       if browser_ua = user_agent.match(/^([^\/]+)\/([^\/\s]+)/)
-        { os: browser_ua[1], editor: browser_ua[2], err: nil }
+        if user_agent.include?("wakatime") then
```

Comment:
Thanks for this! Normally I'm not big on tests, but could we update the tests for this service to include some examples to show this sort of behavior?